refinery/file_store/migrations/0003_higlass_filetypes_and_fileextensions.py

```diff
@@ -0,0 +1,82 @@
+# encoding: utf8
+from django.db import migrations, transaction, IntegrityError
```

Comment:
@jkmarx migrations are excluded from our flake8 checks because the auto generated files don't adhere to PEP8